Sources/Sentry/SentryTracer.m

```diff
@@ -608,7 +614,10 @@ - (nullable SentryAppStartMeasurement *)getAppStartMeasurement
 
     [appStartSpan setTimestamp:appStartEndTimestamp];
 
-    return @[ appStartSpan, runtimeInitSpan, appInitSpan, frameRenderSpan ];
+    NSString *message = [NSString stringWithFormat:@"Phil: %@", appStartMeasurement.description];
+    [SentryLog logWithMessage:message andLevel:kSentryLevelInfo];
```

Comment:
What is this "Phil"?

Reply:
Ups, removed it.

Reply:
Phil is a prefix to find my debug log messages 😁